modules/iam/src/main/scala/ch/epfl/bluebrain/nexus/commons/iam/identity/Identity.scala

```diff
@@ -17,39 +24,143 @@ trait Authenticated
 
 object Identity {
 
+  val authenticatedKey     = "authenticated"
+  val anonymousKey         = "anonymous"
+  private val allowedInput = "([^/]*)"
```

Comment:
Groups cannot start with slash. It will create a wrong URI. I prevent that on the `UserInfo` `Decoder`

Reply:
I think we shouldn't modify the group/sub names, URL encoding them when we store them as part of `@id` would be a better option imo

Reply:
That's in any case the decision of the client when using the `Identity` and not the concert of `Identity` itself, since the `IdentityId` can be restricted to whatever regex we want.